### PR TITLE
[Keyboard] New Keyboard - Waka60

### PR DESCRIPTION
**Pull request by kylemccreery**
## Description

Initial PR for adding the Waka60 keyboard kit by MechWild. This is the default, VIA, and audio enabled keymaps.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechwild/waka60/config.h [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+/*== or choose animations ==*/
+//#    define RGBLIGHT_EFFECT_BREATHING
+//#    define RGBLIGHT_EFFECT_RAINBOW_MOOD
+//#    define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+//#    define RGBLIGHT_EFFECT_SNAKE
+//#    define RGBLIGHT_EFFECT_KNIGHT
+//#    define RGBLIGHT_EFFECT_CHRISTMAS
+//#    define RGBLIGHT_EFFECT_STATIC_GRADIENT
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
```
Comment (drashna) [inline keyboards/mechwild/waka60/config.h:80]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/mechwild/waka60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/mechwild/waka60/rules.mk:4]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/mechwild/waka60/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Waka60
+
+![waka](https://i.ibb.co/10cG7G2/image.png)
```
Comment (fauxpark) [inline keyboards/mechwild/waka60/readme.md:3]: Please rehost this on imgur, and add `h` to the end of the filename.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mechwild/waka60/readme.md (hunk 3 above)
Comment (kylemccreery) [reply keyboards/mechwild/waka60/readme.md]: Changed. When we uploaded that file there was some kind of drama regarding imgur and the permissions/longevity of the files that they were hosting.

**Comment by kylemccreery** [comment]
@fauxpark Just checking if those changes that were made were adequate for what you expected.

File: keyboards/mechwild/waka60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/mechwild/waka60/rules.mk:12]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**